packages/core/admin/admin/src/components/LeftMenu/index.js

```diff
@@ -125,7 +126,23 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
               const Icon = link.icon;
 
               return (
-                <NavLink as={RouterNavLink} to={link.to} key={link.to} icon={<Icon />}>
+                <NavLink
+                  as={RouterNavLink}
+                  to={link.to}
+                  key={link.to}
+                  icon={<Icon />}
+                  isActive={(match, location) => {
```

Comment:
won't this fire every render?

Reply:
Also: what does this number tells you in the end? Wouldn't it make more sense to track the pageviews of the CTB?

Reply:
The PM wants to track every click in the Content-Type-Builder link inside the LeftMenu

Reply:
It's not simply a matter of counting the clicks on the CTB link in the sidebar, but having a starting point to track the funnels of users interacting with the CTB. It already exists for other items in the sidebar, like didGoToMarketplace.

Reply:
@marcoautiero Ok, makes sense. What do you think about adding a generic event then, which tracks a navigation in the menu and sends the page as payload? This way we have data for all interactions and don't have to introduce different events ...

Reply:
Yep, I thought about it, a `didNavigate` with `from` and `to` properties

Reply:
I think I'd prefer that if that is an option :)

Reply:
I agree

Reply:
Making a check with Aurélien

Reply:
I received Aurélien approval on the `didNavigate`, sorry Simone, we need to change the description of the User story and the code :) But the future generations will be grateful to us u.u

Reply:
ok sure, can you please also update the ticket with the new requirements, thanks